bunq/sdk/exception_factory.py

```diff
@@ -21,40 +21,86 @@ class ExceptionFactory:
 
     # Constants for formatting messages
     _FORMAT_RESPONSE_CODE_LINE = 'HTTP Response Code: {}'
-    _GLUE_ERROR_MESSAGES = '\n'
+    _FORMAT_RESPONSE_ID_LINE = 'The response id to help bunq debug: {}'
+    _FORMAT_ERROR_MESSAGE_LINE = 'Error message: {}'
+    _GLUE_ERROR_MESSAGE_NEW_LINE = '\n'
+    _GLUE_ERROR_MESSAGE_STRING_EMPTY = ''
 
     @classmethod
-    def create_exception_for_response(cls, response_code, messages):
+    def create_exception_for_response(
```

Comment:
Arg list not formatted according to PEP 8: https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
correct, this is how we do it. The pep8 one is weird and ugly.

Reply:
Agreed 😝